src/test/java/xjs/data/serialization/writer/JsonWriterTest.java

```diff
@@ -52,6 +52,12 @@ public void write_printsDecimal() {
         assertEquals("12.34", write(Json.value(12.34)));
     }
 
+    @Test
+    public void write_printsInfiniteNumbers() {
```

Comment:
`Infinity` is not a reserved keyword in JSON format and should not be supported by the JSON serializers.